Makefile

```diff
@@ -565,6 +571,18 @@ clean-docker-images: ## Remove all Grapl images
 		--quiet \
 	| xargs --no-run-if-empty docker rmi --force
 
+clean-docker-volumes: ## Remove all Grapl volumes
```

Comment:
This isn't _all_ our volumes (it doesn't touch the ones for the build containers, for instance), just the ones generated in the course of running `docker-compose` (judging by the label), right? If so, it might be good to reflect that in the target name and the comment (as well as the comment on `DOCKER_VOLUME_FILTER_LABEL`, too).

Reply:
I've updated the comments, however since only the dgraph_export volume is labeled at the moment, I'm pondering whether it makes sense to drop the filter and just blow away all volumes. Thoughts?